### PR TITLE
Update version and ChangeLog for local nimbis release (1.6.5+nimbis.1)

### PR DESCRIPTION
**Pull request by cworth-gh**
This release includes the change we have submitted upstream here:

	https://github.com/nvie/pip-tools/pull/368

### CODE REVIEW

**Comment by cworth-gh** [comment]
<a href="https:&#x2F;&#x2F;trello.com&#x2F;c&#x2F;5eb4vDk0&#x2F;670-use-local-pypi-repository-instead-of-git-urls-in-requirements"> Use local pypi repository instead of git URLS in requirements</a>

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,9 +1,13 @@
+# 1.7.0+nimbis.1
```
Comment (amadornimbis) [inline CHANGELOG.md]: Just a note to other reviewers:

I read this [link](https://www.python.org/dev/peps/pep-0440/#local-version-identifiers) to learn more about local version identifiers. I think it's worth pointing out the limited character set that can be used in these labels.

**Comment by amadornimbis** [comment]
Looks good 👍

File: CHANGELOG.md (hunk 1 above)
Comment (cworth-gh) [inline CHANGELOG.md]: Thanks for pointing that out, Eric. I did read that when coming up with the naming of "nimbis.1" for the local part. But it's good for all of us to know that we only get letters, numbers and the period character here.

**Comment by shaunbrady** [comment]
Are the conflicts with the upstream (which is fine, as we'll just switch to that at some point)?  Else looks good.

**Pull request closed without merging**